builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -501,7 +501,12 @@ def buildLambda(String runtime) {
 		. venv/bin/activate
 		pip install pytest
 		"""
-	}
+	} else if (runtime.indexOf("go") > -1 ){
+    //get all dependencies
+    sh "dep ensure"
```

Comment:
is this command looking at the dependency file and getting all the dependencies?

Reply:
yes